uploader.v2.json

```diff
@@ -1480,6 +1480,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_d"
```

Comment:
@huali027 would you mind to change this spec name to match the actual spec name in your core change, `httpd_conf`. Thanks!

Reply:
> @huali027 would you mind to change this spec name to match the actual spec name in your core change, `httpd_conf`. Thanks!

@bfahr, yes, sure, I'll udpate it in this patch too.